src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/AuditLog.java

```diff
@@ -69,13 +69,15 @@
     void logSSLException(TransportRequest request, Throwable t, String action, Task task);
     void logSSLException(RestRequest request, Throwable t);
 
-    void logDocumentRead(String index, String id, ShardId shardId, Map<String, String> fieldNameValues, ComplianceConfig complianceConfig);
-    void logDocumentWritten(ShardId shardId, GetResult originalIndex, Index currentIndex, IndexResult result, ComplianceConfig complianceConfig);
+    void logDocumentRead(String index, String id, ShardId shardId, Map<String, String> fieldNameValues);
+    void logDocumentWritten(ShardId shardId, GetResult originalIndex, Index currentIndex, IndexResult result);
     void logDocumentDeleted(ShardId shardId, Delete delete, DeleteResult result);
     void logExternalConfig(Settings settings, Environment environment);
     
     // compliance config
     void setComplianceConfig(ComplianceConfig complianceConfig);
+
+    ComplianceConfig getCurrentComplianceConfig();
```

Comment:
`getCurrentComplianceConfig` -> `getComplianceConfig`

Reply:
I thought the name getCurrentComplianceConfig will be more descriptive that config fetched is the latest/current config

Reply:
Let me know if you would like to use `getComplianceConfig` itself or some other name.

Reply:
`getCurrentComplienceConfig()` is confusing, IMO. It is just a getter method, it can only get the current value, there is no any other values to get.